mappings/net/minecraft/predicate/entity/EntityPredicates.mapping

```diff
@@ -41,7 +41,7 @@ CLASS net/minecraft/class_1301 net/minecraft/predicate/entity/EntityPredicates
 		ARG 0 entity
 	METHOD method_5913 rides (Lnet/minecraft/class_1297;)Ljava/util/function/Predicate;
 		ARG 0 entity
-	CLASS class_1302 CanPickup
+	CLASS class_1302 CanEquip
```

Comment:
I'd suggest `Equipable` name instead

Reply:
Alright, I'll change the mappings real quick.